packages/cli/src/services/util.ts

```diff
@@ -73,5 +73,12 @@ export function pathToPosix (relPath: string): string {
   // Windows uses \ rather than / as a path separator.
   // It's important that logical ID's are consistent across platforms, though.
   // Otherwise, checks will be deleted and recreated when `npx checkly deploy` is run on different machines.
-  return relPath.split(path.sep).join(path.posix.sep)
+  return path.normalize(relPath).split(path.sep).join(path.posix.sep).replace(/^C:/, '')
+}
+
+export function splitConfigFilePath (configFile?: string): { configDirectory: string, configFilenames?: string[] } {
+  return {
+    configDirectory: configFile ? pathToPosix(path.join(process.cwd(), path.dirname(configFile))) : process.cwd(),
```

Comment:
This won't work with absolute paths I imagine?

Reply:
@umutuzgur , exactly, only relative path at the moment. I had the 'absolute path support' in mind all during the coding, but it would require more caution and validation before allowing to use it. Maybe we can support it in the future.